### ISSUE TITLE
[Legitimate Site Unblocked] www.aetherscan.tech

### ISSUE THREAD

**Issue by StrixSC** [MetaMask/eth-phishing-detect#11190]
**Domain:**  hXXps[://]www[.]aetherscan[.]tech

**Details:** 
This is the domain of a private EVM based blockchain explorer I am working on, the repo is here https://github.com/StrixSC/Aetherscan.
This is nothing malicious and its mainly just a tool to help Ethereum devs.

- [X] I have checked to make sure that [there is not a duplicate issue](https://github.com/MetaMask/eth-phishing-detect/issues)

**Comment by AlexHerman1** [comment]
do not spam this repo with requests regarding private services/sites.

**Comment by StrixSC** [comment]
> do not spam this repo with requests regarding private services/sites.

Should I just submit a PR with the change myself? I didn't see anyone doing that so I didn't either.

The MetaMask phishing detection triggers whenever I try to access the domain. I opened a previous issue for this under the wrong template and didn't think of changing the name and I closed it instead (sorry for that). It was not my intention to spam.

However, I still need to approve this domain, how should I proceed?
